src/components/Footer/Footer.js

```diff
@@ -24,7 +24,6 @@ import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
 import {COLORS, FONTS} from '../../utils/theme';
 
 const Footer = ({
-  item,
   onPressShare,
   onPressComment,
   totalComment,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `item` prop has been removed from the `Footer` component. Ensure that this prop is not used anywhere else in the component or by any parent components that might be passing it down. If it's still being used, removing it could cause errors.
```diff
-  item,
   onPressShare,
   onPressComment,
   totalComment,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->